content/blog/sleep/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Sleep
+date: "2019-05-08T17:12:03.284Z"
+description: I'll sleep when I'm dead.
+---
+
+> A condition of body and mind which typically recurs for several hours every night, in which the nervous system is inactive, the eyes closed, the postural muscles relaxed, and consciousness practically suspended (Oxford dictionary).
+
+Studies have shown that 1 out of every 2 adults in America is not getting the recommended hours of sleep. 1 out of 3 people is trying to survive on 6 hours or less.
+
+Adults need from 7 to 9 hours of sleep per day and the average American is sleeping **6.5 hours** during the week. This used to be **7.9 hours** in 1942.
+
+There are numbers of important processes happening during the night like muscle repair and grow as well as a body cleanse. There is a sewage system in the brain cleaning the brain of metabolic toxins that we accumulated during the day.
+
+> Like humans, most animals sleep for at least a few hours. Dolphins are interesting because they are conscious breathers and in order to not drown, they need to go to the surface and breathe. They float and shut half of its brain during sleep and keep the other half active to keep the basic body functions active.
+
+`youtube: https://www.youtube.com/watch?v=1ZSzI5mHwS8`
+
+Let's explore what affects sleep.
+
+### What is a circadian rhythm?
+
+It's a 24-hour internal clock, a regulator for sleepiness and wakefulness. When we feel the most energized can vary a bit from person to person, but for most adults, there is a noticeable drop in energy from 1 to 3 in the afternoon and from 2 to 4 at night.
+
+Outside factors like lightness and darkness can also have an impact on the internal clock. The brain sends a signal to the body to release melatonin which makes you sleepy.
+
+### Effects of sleep deprivation
+
+Human beings are the only species that deprive themselves of sleep and we are not wired to compensate for the lack of sleep. Sleep is not like a bank. We can't accumulate a debt and hope to pay it off on the weekend.
+
+![Airport sleep](./airport-sleep.jpg)
+
+Effects of sleep deprivation can show:
+
+- **Worsened brain performance**. Getting less than 6 hours of sleep will drop the time to physical exhaustion by 30%.
+
+- **Lower muscle strength**. The less sleep you have less muscle strength, as well as jump height and running speed decrease.
+
+- **Lack of testesterone**. Men who sleep only 5 to 6 hours a night will have a level of testosterone 6-10 years their senior.
+
+Long term lack of sleep is also connected to diseases, one of them is Alzheimer's disease. This may come as a shocker, but drowsy, sleep deprived driving kills more people on the roads than alcohol or drugs combined.
```

Comment:
_This may come as a shocker, but drowsy, sleep deprived driving kills more people on the roads than alcohol or drugs combined._

this sentence can either be the fifth bullet point or just added bellow as it is now. just replace 'and' with 'or' at the end, because the combination of alcohol and drugs takes both, not just one of the two.